src/test/kotlin/org/rust/RsTestBase.kt

```diff
@@ -212,8 +213,8 @@ abstract class RsTestBase : BasePlatformTestCase(), RsTestCase {
                 error("Can't mix `BothEditions` and `MockEdition` annotations")
             }
             // These functions exist to simplify stacktrace analyzing
-            runTestEdition2015(testRunnable)
             runTestEdition2018(testRunnable)
+            runTestEdition2015(testRunnable)
```

Comment:
BTW looks like we should replace `BothEditions` with `AllEditions` and run tests with all possible editions. Don't insist on doing it in this PR